### ISSUE TITLE
Variable products with more than one attribute are displayed without the attributes

### ISSUE THREAD

**Issue by grigorijurasov** [woocommerce/woocommerce-android#1286]
**Describe the bug**
Variable products made up of more than one attribute are displayed without the attributes

**To Reproduce**
1. Create a variable product with one attribute, create variations from this attribute
2. Purchase it
3. The selected attribute term shows up in the name in the app
4. Add a second attribute, re-create variations
5. Purchase the product (two variation dropdowns should appear on the product page)
6. Only the name of a parent variable product but not the variation itself shows up in the app.

**Screenshots**
1 attribute:
<img width="280" alt="Screenshot 2019-07-26 at 13 37 31" src="https://user-images.githubusercontent.com/13084901/61946195-988a1c80-afaa-11e9-9f69-5d7733aff1f3.png">

2 attributes:

<img width="288" alt="Screenshot 2019-07-26 at 13 38 25" src="https://user-images.githubusercontent.com/13084901/61946252-b6578180-afaa-11e9-96a0-8f32017d2fa6.png">

**Expected behavior**

When a variable product is purchased - I'd expect to see which particular variation is it

**Isolating the problem (mark completed items with an [x]):**
- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**Mobile Environment**
Please include:
- Device: Android simulator
- Android version: 
- WooCommerce Android version: 2.2

**WordPress Environment**
<details>
```
Copy and paste the system status report from **WooCommerce > System Status** in WordPress admin.
```
</details>
https://github.com/woocommerce/woocommerce-ios/issues/1127

**Comment by nbradbury** [comment]
Making some notes for myself here...

Looks like Calypso has this issue, too. Note `wp_admin` shows both attributes for this order (small and blue):

<img width="323" alt="Screen Shot 2019-07-31 at 1 32 05 PM" src="https://user-images.githubusercontent.com/3903757/62233945-9f90a080-b397-11e9-85d7-8115897df9d7.png">

But Calypso, like WCAndroid, only shows small:

<img width="529" alt="calypso" src="https://user-images.githubusercontent.com/3903757/62233946-9f90a080-b397-11e9-8d21-73affd2a66dd.png">

**Comment by nbradbury** [comment]
Putting this on hold since after discussing the issue with Proton, it turns out there's no way using the v3 API to get the variation details on an order. It has been proposed for v4.

**Comment by shiki** [comment]
The necessary REST API changes (https://github.com/woocommerce/woocommerce-rest-api/pull/229) have been merged into Core and will be available with WC 4.7.0.
